conf/core/Testing.conf

```diff
@@ -34,5 +34,7 @@
   "#" : "no way to dynamically change tag merging from within the UI. Since PreserveTypesTagMerger is still experimental, we want to leave",
   "#" : "the merger in the these tests at its original value and possibly change it at a later time.",
   "poi.polygon.tag.merger": "hoot::OverwriteTag2Merger",
+  "#" : "Sort the output tags alphabetically by key for easier reading.",
+  "writer.sort.tags.by.key": "true",
```

Comment:
All unit tests sort the tags for XML output.  This makes for easier debugging when tests fail.